src/bucket/BucketManager.h

```diff
@@ -292,10 +292,17 @@ class BucketManager : NonMovableOrCopyable
     // Scans BucketList for non-live entries to evict starting at the entry
     // pointed to by EvictionIterator. Evicts until `maxEntriesToEvict` entries
     // have been evicted or maxEvictionScanSize bytes have been scanned.
-    void startBackgroundEvictionScan(uint32_t ledgerSeq);
-    void resolveBackgroundEvictionScan(AbstractLedgerTxn& ltx,
-                                       uint32_t ledgerSeq,
-                                       LedgerKeySet const& modifiedKeys);
+    void startBackgroundEvictionScan(uint32_t ledgerSeq, uint32_t ledgerVers);
+
+    // Returns a pair of vectors representing entries evicted this ledger, where
+    // the first vector constains all deleted keys (TTL and temporary), and
+    // the second vector contains all archived keys (persistent and
+    // ContractCode). Note that when an entry is archived, its TTL key will be
+    // included in the deleted keys vector.
+    std::pair<std::vector<LedgerKey>, std::vector<LedgerEntry>>
```

Comment:
nit: I may be overly cautious here, but I feel like it would be safer to return a struct with named fields here. Even though the types are different, there is still some risk of confusion. At least it definitely would make the caller code easier to read.